packages/requisitions/src/RequestRequisition/DetailView/AppBarButtons/UseSuggestedQuantityButton.tsx

```diff
@@ -18,7 +18,7 @@ export const UseSuggestedQuantityButtonComponent = () => {
 
   return (
     <ButtonWithIcon
-      Icon={<EditIcon />}
+      Icon={<ZapIcon />}
```

Comment:
offroad - the edit icon wasn't quite the right fit for the action, which is more of a 'wizard'  thing than an 'edit' thing. Manu had the lightning bolt in his designs, so have taken that through.. here and in the response requisition as well